### PR TITLE
fix shared-as-busy events for owner

### LINKED ISSUE #2787: Calendar entry marked as busy is also busy for creator
Calendar shows only Busy when selecting as Busy. Also for the creating user. The creating user should see the Calendar-Item with full information.

### Steps to reproduce
1. Create an Calendar item in you personal autogenerated calendar
2. Create Name "Test Meeting"
2. set "when shared show only busy"
3. Now the creator can only see "busy" not "Test Meeting". that is incorrect

### Expected behaviour
The creator should see "Test Meeting"

### Actual behaviour
The Calendar entry is changed to busy also for the creator

### Server configuration
Centos 7

**Web server:**
nginx

**Database:**
mariadb

**PHP version:**
php56
**Server version:** (see your admin page)
11.0.0
**Calendar version:** (see the apps page)
1.4.1
**Updated from an older installed version or fresh install:**
update from 10.0.2
**Signing status (ownCloud/Nextcloud 9.0 and above):**
Nextcloud
```
Login as admin user into your cloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
No errors have been found.

**Are you using external storage, if yes which one:** local/smb/sftp/...
local

**Are you using encryption:** yes/no
no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
ldap

### Client configuration
**Browser:**
Chrome, Firefox

**Operating system:**
Windows, Lnux

**CalDAV-clients:**

### PR DESCRIPTION
**Pull request by georgehrke**
fixed #2787

### CODE REVIEW

**Comment by georgehrke** [comment]
please review @MorrisJobke @nickvergessen

**Review by nickvergessen — APPROVED** [review]
Works

**Comment by nickvergessen** [comment]
Should backport

File: apps/dav/lib/CalDAV/CalendarObject.php [hunk 1]
```diff
@@ -42,7 +44,11 @@ function get() {
 	}
 
 	private function isShared() {
-		return isset($this->calendarInfo['{http://owncloud.org/ns}owner-principal']);
+		if (!isset($this->calendarInfo['{http://owncloud.org/ns}owner-principal'])) {
+			return false;
```
Comment (LukasReschke) [inline apps/dav/lib/CalDAV/CalendarObject.php]: Not covered by a test 😉

**Review by LukasReschke — APPROVED** [review]

**Comment by georgehrke** [comment]
@karlitschek 👍 for backporting?

**Comment by karlitschek** [comment]
please backport

**Comment by nickvergessen** [comment]
Also backport to 10?

**Comment by georgehrke** [comment]
@nickvergessen No, this was related to the activity-hooks added in nextcloud 11

**Pull request merged**